python/s3_bucket_default_encryption_enabled.py

```diff
@@ -0,0 +1,106 @@
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+#
+# Description: Check that S3 buckets have default encryption enabled.
+#
+# Trigger Type: Change Triggered
+# Scope of Changes: S3:Bucket
+# Accepted Parameters: None
+# Your Lambda function execution role will need to have a policy that provides
+# the appropriate permissions. Here is a policy that you can consider.
+# You should validate this for your own environment.
+#
+# {
+#    "Version": "2012-10-17",
+#    "Statement": [
+#        {
+#            "Effect": "Allow",
+#            "Action": [
+#                "logs:CreateLogGroup",
+#                "logs:CreateLogStream",
+#                "logs:PutLogEvents"
+#            ],
+#            "Resource": "arn:aws:logs:*:*:*"
+#        },
+#        {
+#            "Effect": "Allow",
+#            "Action": [
+#                "config:PutEvaluations"
+#            ],
+#            "Resource": "*"
+#        },
+#        {
+#            "Effect": "Allow",
+#            "Action": [
+#                "s3:Get*"
```

Comment:
Action is to permissive.  Can it be scoped down to "s3:GetEncryptionConfiguration"